server/src/metadata/field-metadata/hooks/before-create-one-field.hook.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+
+import {
+  BeforeCreateOneHook,
+  CreateOneInputType,
+} from '@ptc-org/nestjs-query-graphql';
+
+import { FieldMetadata } from 'src/metadata/field-metadata/field-metadata.entity';
+
+@Injectable()
+export class BeforeCreateOneField<T extends FieldMetadata>
```

Comment:
Nestjs-query hooks might not be the best to do that, I'm open to suggestions. I wanted to experiment with them anyway